_data/projects/learn-js.yml

```
@@ -0,0 +1,12 @@
+name: LearnJS
+desc: A website having collection of javascript programs and questions. 
+site: https://github.com/akulsr0/learn-js
+tags:
+  - javascript
+  - programs
+  - questions
+  - algorithm
+  - data-structures
+upforgrabs:
+  name: up for grabs
+  link: https://github.com/akulsr0/learn-js/labels/up%20for%20grabs
```

And if `help wanted` is the right label, this should be the corresponding URL:

```suggestion
  link: https://github.com/akulsr0/learn-js/labels/help%20wanted
```

---

Created a new PR!

---

@akulsr0 for next time you're working on a PR, you can continue to push commits to your branch after opening the PR and it'll synchronize the changes - no need to start fresh, and makes reviewing easier.